<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STDAtmo</class>
 <widget class="QMainWindow" name="STDAtmo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>STDAtmo</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>371</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Calculator</string>
     </attribute>
     <widget class="QComboBox" name="TempUnits">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>°C / K</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>°F / °R</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="AltUnits">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>m</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ft</string>
       </property>
      </item>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="Input">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>150</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="text">
       <string>Input:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Input_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>150</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="text">
       <string>Output:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Compute">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>81</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Compute</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Reset">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>130</y>
        <width>81</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>63</y>
        <width>152</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Geometric altitude</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Temperature deviation</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="AltitudeInp">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>79</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="dISAInp">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>79</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>271</x>
        <y>211</y>
        <width>87</width>
        <height>240</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="GopAltCombo">
         <item>
          <property name="text">
           <string>m</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ft</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="PressureCombo">
         <item>
          <property name="text">
           <string>Pa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>psi</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mm Hg</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="TemperatureCombo">
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>°C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>°F</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>°R</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="DensityCombo">
         <item>
          <property name="text">
           <string>kg/m3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>slug/m3</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="SpeedSoundCombo">
         <item>
          <property name="text">
           <string>m/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ft/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>km/hr</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mi/hr</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>knots</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ViscosityCombo">
         <item>
          <property name="text">
           <string>Pa.s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>lbf.s/ft2</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="GravityCombo">
         <item>
          <property name="text">
           <string>m/s2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ft/s2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>181</x>
        <y>211</y>
        <width>81</width>
        <height>240</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="AltitudeOutput">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PressureOutput">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="TemperatureOutput">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DensityOutput"/>
       </item>
       <item>
        <widget class="QLineEdit" name="SoundSpeedOutput"/>
       </item>
       <item>
        <widget class="QLineEdit" name="ViscosityOutput"/>
       </item>
       <item>
        <widget class="QLineEdit" name="GravityOutput"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>211</y>
        <width>152</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Geopotential altitude</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Pressure</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Temperature</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Density</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Speed of sound</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Dynamic viscosity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Acceleration of gravity</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Plots</string>
     </attribute>
     <widget class="QCustomPlot" name="PlottingWidget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>341</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="AltitudeMinInp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="AltitudeMaxInp">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="UnitTypePlot">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Temperature</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pressure</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Density</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Speed of sound</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dynamic viscosity</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Acceleration of gravity</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="minAltUnit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>m</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>1</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Min altitude</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>83</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Max altitude</string>
      </property>
     </widget>
     <widget class="QLabel" name="maxAltUnit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>m</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>-10</y>
        <width>89</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Increment</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="dAltUnit">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>m</string>
      </property>
     </widget>
     <widget class="QComboBox" name="UnitPlot">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>K</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>°C</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="DAltInp">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="UnitSetup">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>51</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>SI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>US</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="Plot_graph">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>61</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Info</string>
     </attribute>
     <widget class="QTextBrowser" name="infoBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>361</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
